gravitee-apim-console-webui/src/management/api/runtime-logs-v4/runtime-logs/components/api-runtime-logs-quick-filters/pipes/chip-value.pipe.ts

```diff
@@ -17,18 +17,26 @@ import { Pipe, PipeTransform } from '@angular/core';
 
 import { MultiFilter, SimpleFilter } from '../../../models';
 
+type ArrayFilter = MultiFilter | string[] | Set<number>;
+
 @Pipe({ name: 'chipValue' })
 export class ChipValuePipe implements PipeTransform {
-  transform(filter: SimpleFilter | MultiFilter | string[]): string {
-    return Array.isArray(filter)
-      ? filter
-          .map((value) => {
-            if (value.label) {
-              return value.label;
-            }
-            return value;
-          })
-          ?.join(', ')
-      : filter.label;
+  transform(filter: SimpleFilter | ArrayFilter): string {
+    if (Array.isArray(filter)) {
+      return filter
+        .map((value) => {
+          if (value.label) {
+            return value.label;
+          }
+          return value;
+        })
+        ?.join(', ');
+    }
+
+    if (filter instanceof Set) {
+      return Array.from(filter)?.join(', ');
+    }
```

Comment:
I would have done that before the first if, and just returning the array from filter
so you keep the logic of joining at only one place (so if someday the separator changes we avoid to forget to replace the other one too)